packages/firestore/src/core/firestore_client.ts

```diff
@@ -108,7 +108,7 @@ export class FirestoreClient {
   private lruScheduler?: LruScheduler;
 
   private readonly clientId = AutoId.newId();
-  private isShutdown = false;
+  private _isShutdown = false;
```

Comment:
Optional: You might consider dropping the "is" prefix for the name. Other boolean properties in the source tree are typically named without it (e.g. started, committed, current).